src/test/java/bogdanSandu/tests/Tema7/CURAHealthcare.java

```diff
@@ -0,0 +1,78 @@
+package bogdanSandu.tests.Tema7;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class CURAHealthcare extends TestBase {
+
+    @DataProvider(name = "data-set")
+    public Object[][] loginDataProvider(){
+        return new Object[][]{
+                {"valid", "John Doe", "ThisIsNotAPassword",},
+                {"invalid", "John Do", "ThisIsNotAPassword",}
+        };
+    }
+
+    @Test(dataProvider = "data-set")
+    public void login(String type, String username, String password) {
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(3));
+        driver.findElement(By.id("menu-toggle")).click();
+        System.out.println("Click on menu");
+        driver.findElement(By.xpath("//*[@id=\"sidebar-wrapper\"]/ul/li[3]/a")).click();
+        System.out.println("Click on login menu");
+        driver.findElement(By.id("txt-username")).sendKeys(username);
+        driver.findElement(By.id("txt-password")).sendKeys(password);
+        driver.findElement(By.id("btn-login")).click();
+        if(type.equals("valid")){
+            wait.until(ExpectedConditions.presenceOfElementLocated(
+                    By.cssSelector("#appointment > div > div > div > h2")));
+        }else
+            wait.until(ExpectedConditions.presenceOfElementLocated(
+                    By.cssSelector("#login > div > div > div.col-sm-12.text-center > p.lead.text-danger")));
+    }
+
+    public void completeDetailsForAppointment(){
+        driver.findElement(By.id("txt-username")).sendKeys("John Doe");
+        driver.findElement(By.id("txt-password")).sendKeys("ThisIsNotAPassword");
+        driver.findElement(By.id("btn-login")).click();
+    }
+
+    @Test(dataProvider = "data-set")
+    public void makeAppointment(String type, String username, String password){
```

Comment:
Please double check the requirement. In order to be on appointment page you need to be logged in -> Se vor testa diferite scenarii pozitive și negative de rezervare (măcar 1 scenariu pozitiv și un scenariu negativ) Thanks!